Styles/style.css

```diff
@@ -0,0 +1,976 @@
+    color: #333;
+  }
+
+  .sponsors-wrap {
+    text-align: center;
+    background-color: #2e2e2e;
+  }
+
+  footer .content {
+    text-align: center;
+    padding: 16px 0;
+    width: 90%;
+    margin: auto;
+  }
+
+  .sponsors-wrap .content {
+    text-align: center;
+    padding-top: 80px;
+    padding-bottom: 45px;
+    color: white;
+    height: auto;
+  }
+
+  .sponsors-wrap .content .sponsor-logos {
+    overflow: auto;
+    height: auto;
+    margin: auto;
+    padding: 0;
+    vertical-align: middle;
+    text-align: center;
+    width: 70%;
+    position: relative;
+  }
+
+  footer {
+    font-size: 0.87em;
+    color: #272a32;
+  }
+
+  .wrap {
+    width: unset;
+    overflow-x: hidden;
+  }
+
+  .wrap > nav,
+  .wrap div.content {
+    max-width: 1100px;
+    width: 93%;
+    margin: 0 auto;
+  }
+
+  .about-landing-wrap {
+    height: auto;
+  }
+
+  .mobile_menu_button {
+    position: absolute;
+    top: 35px;
+    left: 20px;
+    z-index: 1;
+    width: 32px;
+  }
+
+  .about-landing-wrap div.content {
+    padding: 18px;
+    text-align: left;
+    align-items: flex-start;
+  }
+
+  .about-landing-wrap div.content img.title-image {
+    width: 90%;
+    padding-top: 120px;
+  }
+
+  .about-landing-wrap div.content .header-card {
+    width: unset;
+    text-align: center;
+    margin: 0 auto 33px;
+    padding: 32px;
+    line-height: 25px;
+    border: 1px solid #dfdfdf;
+    background-color: #fff;
+    font-size: 0.875em;
+  }
+
+  .logo-wrapper .content {
+    text-align: center;
+    padding: 48px 12px 36px;
+  }
+
+  .summits-wrapper .content {
+    padding: 48px 12px 36px;
+    text-align: center;
+  }
+
+  footer.dark_footer {
+    background-color: #2e2e2e;
+  }
+
+  footer.dark_footer div.content {
+    color: white;
+  }
+}
+
+@media only screen and (max-width: 300px) {
+  .program-info-wraper section .program-card {
+    width: 93%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 8px;
+    padding: 16px 0;
+    height: auto;
+    overflow: auto;
+    background: rgba(255, 255, 255, 0.1);
+  }
+
+  .program-info-wraper section .program-card .program-sub-description {
+    width: calc(100% - 5em);
+    float: left;
+    font-size: 0.81em;
+    text-align: left;
+    word-break: keep-all;
+    overflow: auto;
+    height: auto;
+    color: #fff;
+    line-height: 1.5em;
+    padding: 8px 17px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  html {
+    overflow-y: scroll;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  img {
+    border: 0;
+  }
+
+  p {
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .toggle-menu {
+    display: none;
+  }
+
+  .mobile_menu_button {
+    display: none;
+  }
+
+  body {
+    display: unset;
+    margin: 0;
+    padding: 0;
+    font-family: nanum-gothic, sans-serif;
+    color: rgb(51, 51, 51);
+  }
```

Comment:
This part of the code is completely repeating from the beginning of this CSS file. if you remove that redundant media query for mobile, you won't have to repeat this code again. 

- [ ] Kindly, take into consideration removing all the repeating code you have in this 2 media queries;

Reply:
Thank you. scape plan with the media section up to 300px.